F/FLAC/build_tarballs.jl

```diff
@@ -20,6 +23,14 @@ cd $WORKSPACE/srcdir/flac-*/
 # kernel headers, rather than the glibc headers, sicne our glibc is too old
 atomic_patch -p1 "${WORKSPACE}/srcdir/patches/flac_linux_headers.patch"
 
+if [[ "${target}" == *-mingw* ]]; then 
+    # Fix error 
+    #    sed/sed-compile.o: In function `sprintf': 
+    #    /opt/x86_64-w64-mingw32/x86_64-w64-mingw32/sys-root/include/stdio.h:366: undefined reference to `__chk_fail' 
```

Comment:
Please use the actual error message found here, otherwise it's confusing and it doesn't help with searching similar errors
```suggestion
    #     .libs/metadata_iterators.o:metadata_iterators.c:(.text+0x106b): undefined reference to `__memset_chk'
```